contracts/RewardManager.sol

```diff
@@ -86,7 +91,8 @@ contract RewardManager is Ownable, Versionable, Safe {
     uint256 _rewardeeRegistrationFeeInSPEND,
     uint256 _rewardProgramRegistrationFeeInSPEND,
     address[] calldata _eip1271Contracts,
-    address _versionManager
+    address _versionManager,
+    address _governanceAdmin
```

Comment:
Not a super big deal, but I’ve been trying to keep the version manager address as the last param in the setup function for consistency sake